models/public/mobilenet-v3-large-1.0-224-tf/accuracy-check.yml

```diff
@@ -1,39 +1,11 @@
 models:
-  - name: mobilenet-v3-large-1.0-224-tf
-
-    launchers:
-      - framework: tf
-        model: v3-large_224_1.0_float/v3-large_224_1.0_float.pb
-        adapter: classification
-
-    datasets:
-      - name: imagenet_1001_classes
-        preprocessing:
-          - type: bgr_to_rgb
-          - type: resize
-            size: 256
-          - type: crop
-            size: 224
-          - type: normalization
-            mean: 127.5
-            std: 127.5
-        metrics:
-          - name: accuracy@top1
-            type: accuracy
-            top_k: 1
-            reference: 0.757
-          - name: accuracy@top5
-            type: accuracy
-            top_k: 5
-            reference: 0.9276
-
   - name: mobilenet-v3-large-1.0-224-tf
     launchers:
       - framework: dlsdk
         adapter: classification
 
     datasets:
-      - name: imagenet_1001_classes
+      - name: imagenet_1000_classes
         preprocessing:
           - type: resize
             size: 256
```

Comment:
please update ref numbers in configs too

Reply:
updated

Reply:
we do not plan to compare reference now using AC, configs will be executed using --ignore_reference
but you can send patch (we need update docs with note that ref should be normalized for metrics where it can be used as percentage and use scaling for both value and scale before comparing or compare then apply scale).

P.S. please make these changes as separated PR